ui/app/templates/workspace/projects/project/app/deployment.hbs

```diff
@@ -35,6 +35,16 @@
     </small>
   </div>
   {{/if}}
+  {{#if @model.statusReport.status.completeTime.seconds}}
+    <div class="item">
+      Last checked {{date-format-distance-to-now @model.statusReport.status.completeTime.seconds }}
+    </div>
+  {{/if}}
+  <div class="item">
+    <Pds::Button @variant="ghost" @iconStart="refresh-default" {{on "click"(fn this.refreshHealthCheck)}}>
```

Comment:
[dust] We can drop the `(fn ...)` if we’re not passing any extra args (in fact, the latest version of the linter insists upon it).

```suggestion
    <Pds::Button @variant="ghost" @iconStart="refresh-default" {{on "click" this.refreshHealthCheck}}>
```